builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -452,3 +452,38 @@ def jsonParse(jsonString) {
     def nonLazyMap = new groovy.json.JsonSlurperClassic().parseText(jsonString)
     return nonLazyMap
 }
+
+def updateAclPermission(serviceId, userId) {
+	try {
+		def categoryList = ['code', 'deploy']
+		def policiesList = []
+		for (category in categoryList) {
+			def eachPolicy = [
+				userId: userId,
+				permission: 'write',
+				category: category
+			]
+			policiesList.add(eachPolicy)
+		}
+		echo "policiesList: $policiesList"
+
+		def body = JsonOutput.toJson([
+			serviceId: serviceId,
+			policies: policiesList
+		]);
+		def updatePermission = sh(script: "curl -X POST \
+				${g_base_url}/jazz/acl/policies \
+				-k -v -H \"Authorization: $auth_token\" \
+				-H \"Content-Type: application/json\" \
+				-d \'${body}\'", returnStdout: true).trim()
+		def responseJSON = parseJson(updatePermission)
+
+		if (responseJSON && responseJSON.success) {
```

Comment:
Make sure the response contains "**success**"

Reply:
Done